service/ziti-tunnel/service/ipc.go

```diff
@@ -609,7 +613,7 @@ func toggleIdentity(out *json.Encoder, fingerprint string, onOff bool) {
 		msg := fmt.Sprintf("identity with fingerprint %s not found", fingerprint)
 		log.Warn(msg)
 		respond(out, dto.Response{
-			Code:    SUCCESS,
+			Code:    IDENTITY_NOT_FOUND,
```

Comment:
Clint, I modified this code. If the fingerprint value is not matching, the Identity not found code will be set rather than success. Not sure whether it will affect any other code

Reply:
this makes me think that there should be something in the UI that should be updated?